### ISSUE TITLE
can  you give train image and label for example?

### ISSUE THREAD

**Issue by bear63** [meijieru/crnn.pytorch#7]
I create the lmdb data, and the model is worse。I think the ground truth I set is wrong.

**Comment by meijieru** [comment]
You could see the demo for example.
